packages/pages/package.json

```diff
@@ -86,7 +86,7 @@
     "yaml": "^2.3.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.10",
+    "@babel/core": "^7.23.2",
```

Comment:
Hmm, do we even need babel anymore? I think it was necessary when using Jest but I removed that. It's possible I just missed this. Try removing it completely and testing things still work.

Reply:
Oh ur correct, it's unnecessary